plugin-dev/Source/Sentry/Public/SentrySubsystem.h

```diff
@@ -268,6 +268,10 @@ class SENTRY_API USentrySubsystem : public UEngineSubsystem
 	UFUNCTION(BlueprintCallable, Category = "Sentry")
 	USentryTransaction* StartTransactionWithContextAndOptions(USentryTransactionContext* Context, const TMap<FString, FString>& Options);
 
+	bool IsSupportedForCurrentSettings()
+	{
+		return IsCurrentBuildConfigurationEnabled() && IsCurrentBuildTargetEnabled() && IsCurrentPlatformEnabled();
```

Comment:
I recall we've added an `EnableForPromotedBuildsOnly` flag along with these settings - shouldn't it also be checked here?

Reply:
@tustanivsky this was updated btw.

Reply:
I think we don't have the `USentrySubsystem::EnableForPromotedBuildsOnly()` method at this moment - is this something specific to your fork repo?

Reply:
No worries, apparently I've misplaced my previous comment.